### PR TITLE
fix: Modify v1 UpgradeHandler to correctly execute migration [DEV-1973]

### PR DESCRIPTION
**Pull request by Eengineer1**
- Fixes ICA module store key initialisation.

### CODE REVIEW

**Review by ankurdotb — CHANGES_REQUESTED** [review]
1. Send exact details of what was migrated _if_ the node's logging mode is set to debug mode. This should include previous data _and_ new data. Skip Protobuf migrations and do it for everything else.
2. Query the module version map at v0.6.9 **before** upgrade as well (no need for earlier upgrade sequences). This should match what is currently on mainnet before trying to upgrade to v1.0.x

File: app/const.go [hunk 1]
```diff
@@ -5,4 +5,28 @@ const (
 	Home = ".cheqdnode"
 
 	UpgradeName = "v1.0.0"
```
[the remark this answers is not in the document]
Comment (Eengineer1) [reply app/const.go:7]: Added.

**Comment by ankurdotb** [comment]
Task linked: [DEV-1973 Fix upgrade handler for v1.0.0 migrations](https://app.clickup.com/t/6600954/DEV-1973)

**Review by ankurdotb — CHANGES_REQUESTED** [review]
LGTM!

File: app/app.go [hunk 2]
```diff
@@ -521,8 +609,11 @@ func New(
 		groupmodule.NewAppModule(appCodec, app.GroupKeeper, app.AccountKeeper, app.BankKeeper, app.interfaceRegistry),
 		ibc.NewAppModule(app.IBCKeeper),
 		params.NewAppModule(app.ParamsKeeper),
-		transferModule,
+		// IBC modules
+		transfer.NewAppModule(app.TransferKeeper),
+		ibcfee.NewAppModule(app.IBCFeeKeeper),
 		icaModule,
```
Comment (ankurdotb) [inline app/app.go:615]: Does this one not need an `app.<x>Keeper` call?

**Review by ankurdotb — COMMENTED** [review]
There are a lot of comments in here for InterChain accounts which I'm not 100% sure whether necessary. Also, added a review comment on one specific bit regarding keepers.

**Review by ankurdotb — APPROVED** [review]

**Pull request merged**